### ISSUE TITLE
same email for two persons

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#368]
preprodpp 0.3.27

Was able to set the same email for two persons. I think we validated for that in a previous version? So this should not be able:

![screen](https://cloud.githubusercontent.com/assets/1533081/7203763/5ae7aac0-e51d-11e4-8622-5893cece6c36.png)

**Comment by smadamit** [comment]
At the end of the checkout it results in this bug:

```
An exception occurred while executing 'INSERT INTO users (username, email, displayName, firstname, surname, Country_id, password, hashkey, active, birthday, updated, created) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [null, "tim.adams.30@jimev.de", null, "Leonie", "Adams", 81, null, null, 1, "1990-10-19", "2015-04-17 16:21:21", "2015-04-17 16:21:21"]:

SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'tim.adams.30@jimev.de' for key 'UNIQ_1483A5E9E7927C74'
```

**Comment by inbaz** [comment]
fixed in 0.2.28
